vignettes/methodological_introduction.Rmd

```diff
@@ -0,0 +1,109 @@
+\]
+comprises a random-effects component, given by $Z_i \Psi Z_{i}^{T}$, and a within-subject component, given by $\Delta_i$. 
+When fitting such models, there will generally be a "competition" and trade-off between the complexities of the two components, and care must be exercised to prevent nonidentifiability, or near nonidentifiability, of the parameters. This is one of the reasons why it can be advantageous in practice to use only one of these components as long as it still allows to capture all relevant sources of variability. In longitudinal studies with only one level of grouping, the within-subject component is particularly important to be considered in order to account for within-subject correlation, whereas an additional random-effects component is often not strictly needed.
+
+Of note, in the literature, the random-effects component and the within-subject component are sometimes also referred to as $R$ and $G$, or as $R$-side and $R$-side random effects (@Cnaan1997, @Littell2000).
+
+## The MMRM as a special case
+
+In a clinical trial setting, one often chooses to directly model the variance-covariance structure of the response, i.e. to account for within-subject dependencies using the within-group component $\Delta_i$, and can omit the random effects component ($Z_i b_i$). This yields the MMRM with:
+
+\[
+     y_i = X_i\beta + \epsilon_i, \quad \epsilon_i \sim \mathcal{N}(0,\sigma^2\Delta_i), \quad i=1,\ldots,M.
+\]
+The $\Delta_i$ matrices can be decomposed and be represented by the product of a variance structure component and a correlation structure component,
+
+\[
+    \Delta_i = V_i C_i V_i, 
+\]
+where $V_i$ is a diagonal matrix with all diagonal elements being positive, and $C_i$ is a correlation matrix, i.e.\ a positive definite matrix with all diagonal elements equal to $1$. It holds that
+
+\[
+  \text{Var}(\epsilon_{ij}) = \Sigma = \sigma^2[V_i]_{jj}^2, \quad \text{cor}(\epsilon_{ij}), \epsilon_{ik}= [C_i]_{jk},
+\]
+so that $V_i$ describes the variance and $C_i$ describes the correlation of the within-subject errors $\epsilon_i$.
+This decomposition is appealing both theoretically and computationally, as it allows to model and to estimate the two structures separately and to combine them in a flexible manner.
+
+When modeling longitudinal responses in a clinical trial with multiple follow-up visits, the linear predictor $X_i\beta$ typically considers fixed effects of baseline values, treatment and visit, as well as interactions between treatment and visit, and possibly between baseline and visit. Commonly the fixed effects are of most interest and the correlation structure $\sigma^2\Delta_i$ can be viewed as a nuisance quantity. However, it is important to model it carefully, since it affects validity of the estimated variance of $\beta$. The `mmrm`-package supports a wide range of correlation structures and also makes use of the decomposition.
```

Comment:
```suggestion
When modeling longitudinal responses in a clinical trial with multiple follow-up visits, the linear predictor $X_i\beta$ typically considers fixed effects of baseline values, treatment and visit, as well as interactions between treatment and visit, and possibly between baseline and visit. Commonly the fixed effects are of most interest and the correlation structure $\sigma^2\Delta_i$ can be viewed as a nuisance quantity. However, it is important to model it carefully, since it affects validity of the estimated variance of $\beta$. The `mmrm` package supports a wide range of correlation structures and also makes use of the decomposition.
```